steps/check-your-appeal/template.html

```diff
@@ -1,2 +1,22 @@
 {% extends "components/globals.html" %}
 {% extends "look-and-feel/layouts/check_your_answers.html" %}
+{% from "look-and-feel/components/fields.njk" import textbox, formSection %}
+
+{% set pageContent = {
+    sendApplication: content.submitButton.value
+    }
+%}
+
+{% block statement_of_truth_content %}
+    {{ header('Sign and submit', size='medium') }}
+    <p>The information I have provided in this appeal is accurate, to the best of my knowledge.</p>
+    <p>I give permission for you to correspond with my named representative about my appeal.</p>
+    <p>I agree to the <a href="/terms-and-conditions">terms and conditions</a> of using the ‘Appeal a benefit decision’ service.</p>
```

Comment:
Please move all this content out into a content file for i18n purposes.

Reply:
done